app/views/images/index.html.erb

```diff
@@ -3,35 +3,37 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+    <div class="app-pane">
+      <% upload_image_heading = capture do %>
+        <h2 class="govuk-heading-m">
+          <%= t("images.index.upload_image") %>
+        </h2>
+      <% end %>
 
-    <% unless rendering_context == 'modal' %>
-      <div class="app-pane">
-        <% upload_image_heading = capture do %>
-          <h2 class="govuk-heading-m">
-            <%= t("images.index.upload_image") %>
-          </h2>
-        <% end %>
-
-        <%= form_tag(
-          create_image_path(@document),
-          multipart: true,
-          data: { gtm: "image-upload-submit" },
-        ) do %>
-          <%= render "govuk_publishing_components/components/file_upload", {
-            label: {
-              text: upload_image_heading,
-            },
-            name: "image"
-          } %>
-
-          <%= render_govspeak(t("images.index.description_govspeak")) %>
-
-          <%= render "govuk_publishing_components/components/button", {
-            text: "Upload",
-          } %>
-        <% end %>
-      </div>
-    <% end %>
+      <%= form_tag(
+        create_image_path(@document),
+        id: "image-upload",
+        multipart: true,
+        data: { gtm: "image-upload-submit" },
```

Comment:
Do you know if GTM will still receive this event? I assume this listens to form submit events but it doesn't look like we listen to any of those.

Reply:
@alex-ju is currently working on rewriting our local `modules/` to support re-initialisation, just like components. I think you're referring to `GTMFormListener` here, so it should be fixed by that :-).

Reply:
I was referring to lack of a submit event existing. If GTM is listening for form submit events then this one will never be called because we preventDefault on the button click event.

GTM likely has all sorts of voodoo though so this may not be remotely a problem.

Reply:
You're right - it wouldn't have worked. I've just added a commit that, with the work @alex-ju is doing, will preserve the current behaviour.